tests/test_typings.py

```diff
@@ -11,13 +11,16 @@
 class MyPyTest(TestCase):
     """Class for testing modules with mypy."""
 
+
+    @unittest.skip("Causing out of memory error on travis")
```

Comment:
could this and the following skipped method be added to a separate suite that is run by default but not on travis e.g., maybe some form of integration test or just "non-travis-tests"? maybe worth thinking about how to handle this kind of situation in general